flows/definition/legacy.go

```diff
@@ -563,15 +569,18 @@ func createRuleNode(lang utils.Language, r legacyRuleSet, translations *flowTran
 	node := &node{}
 	node.uuid = r.UUID
 
+	baseRouter := routers.BaseRouter{ResultName_: r.Label}
+
 	exits, cases, defaultExit := parseRules(lang, r, translations)
 
 	switch r.Type {
 	case "subflow":
 		// subflow rulesets operate on the child flow status
 		node.router = &routers.SwitchRouter{
-			Default: defaultExit,
-			Operand: "@child.status",
-			Cases:   cases,
+			BaseRouter: baseRouter,
```

Comment:
Passing in the base class like this is probably starting to get to the point where we should have a constructor instead.. ie, `NewSwitchRouter()`